extensions/visualizations/oppia-visualization-enumerated-frequency-table.directive.spec.ts

```diff
@@ -20,6 +20,8 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { ComponentFixture, waitForAsync, TestBed, fakeAsync, tick } from '@angular/core/testing';
 import { AnswerStats } from 'domain/exploration/answer-stats.model';
+// TODO(sll): Remove this once the directive is migrated to Angular.
```

Comment:
This does not look right, should it follow the TODO format used in other places?